zone/attack.cpp

```diff
@@ -4029,27 +4030,38 @@ void Mob::CommonDamage(Mob* attacker, int64 &damage, const uint16 spell_id, cons
 				range = RuleI(Range, DamageMessages);
 			}
 
-			entity_list.QueueCloseClients(
-				this, /* Sender */
-				outapp, /* packet */
-				true, /* Skip Sender */
-				range, /* distance packet travels at the speed of sound */
-				(IsValidSpell(spell_id)) ? 0 : skip, /* Skip if not spell */
-				true, /* Packet ACK */
-				filter /* eqFilterType filter */
-				);
-
-			// Send damage to client (including inates skill_id)
-			if (IsClient()) {
-				//I dont think any filters apply to damage affecting us
-				CastToClient()->QueuePacket(outapp);
+			// If an "innate" spell, change to spell type to
+			// produce a spell message.  Send to everyone.
+			// This fixes issues with npc-procs like 1002 and 918 which
+			// need to spit out extra spell color.
+			if (IsValidSpell(spell_id) && skill_used == 52) {
+				a->type = DamageTypeSpell;
+				entity_list.QueueCloseClients(
+					this, /* Sender */
+					outapp, /* packet */
+					false, /* Skip Sender */
+					range, /* distance packet travels at the speed of sound */
+					0, /* don't skip anyone on spell */
+					true, /* Packet ACK */
+					filter /* eqFilterType filter */
+					);
 			}
+			else {
+				//I dont think any filters apply to damage affecting us
+				if (IsClient()) {
+					CastToClient()->QueuePacket(outapp);
+				}
 
-			// If the "spell" was an "innate", change to spell type to
-			// produce spell message as well.
-			if (IsClient() && IsValidSpell(spell_id) && skill_used == 52) {
-				a->type = DamageTypeSpell;
-				CastToClient()->QueuePacket(outapp);
+				// Otherwise, send normal spell or melee message to observers.
+				entity_list.QueueCloseClients(
+					this, /* Sender */
+					outapp, /* packet */
+					true, /* Skip Sender */
+					range, /* distance packet travels at the speed of sound */
+					(IsValidSpell(spell_id) && skill_used != 52) ? 0 : skip,
```

Comment:
EQ::skills::SkillTigerClaw

Reply:
Yeah - weird right.  This is what is in skill field in spells_new for spells like 1002....

Reply:
Do we really want to use that constant or something that desribes it diff?

Reply:
The constant is descriptive enough, it's just a skill ID and that's all we need to know versus a magic "52".

Reply:
Done!